src/views/Viewer.vue

```diff
@@ -26,7 +26,7 @@
 		v-if="initiated || currentFile.modal"
 		id="viewer"
 		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed}"
-		:clear-view-delay="isTesting ? -1 : 5000 /* prevent cypress timeouts */"
+		:clear-view-delay="isTesting || isMobile ? -1 : 5000 /* prevent cypress timeouts and disable on mobile */"
```

Comment:
We should maybe avoid specifying 500 and use the default from the `Modal` component (maybe passing undefined works here?)

Reply:
I'll try

Reply:
works